cmd/explaintest/r/black_list.result

```diff
@@ -5,52 +5,52 @@ explain select * from t where a < 1;
 id	count	task	operator info
 TableReader_7	3323.33	root	data:Selection_6
 └─Selection_6	3323.33	cop[tikv]	lt(Column#1, 1)
-  └─TableScan_5	10000.00	cop[tikv]	table:t, range:[-inf,+inf], keep order:false, stats:pseudo
+  └─TableScan_5	10000.00	cop[tikv]	table:t, mapping:a->#1, range:[-inf,+inf], keep order:false, stats:pseudo
 insert into mysql.opt_rule_blacklist values('predicate_push_down');
 admin reload opt_rule_blacklist;
 
 explain select * from t where a < 1;
 id	count	task	operator info
 Selection_5	8000.00	root	lt(Column#1, 1)
 └─TableReader_7	10000.00	root	data:TableScan_6
-  └─TableScan_6	10000.00	cop[tikv]	table:t, range:[-inf,+inf], keep order:false, stats:pseudo
+  └─TableScan_6	10000.00	cop[tikv]	table:t, mapping:a->#1, range:[-inf,+inf], keep order:false, stats:pseudo
 delete from mysql.opt_rule_blacklist where name='predicate_push_down';
 admin reload opt_rule_blacklist;
 
 explain select * from t where a < 1;
 id	count	task	operator info
 TableReader_7	3323.33	root	data:Selection_6
 └─Selection_6	3323.33	cop[tikv]	lt(Column#1, 1)
-  └─TableScan_5	10000.00	cop[tikv]	table:t, range:[-inf,+inf], keep order:false, stats:pseudo
+  └─TableScan_5	10000.00	cop[tikv]	table:t, mapping:a->#1, range:[-inf,+inf], keep order:false, stats:pseudo
```

Comment:
Could we replace `Column#1` with something like `a[#1]` or `a(#1)`?

Reply:
I think we need a discussion about the explain format.